modules/appeals_api/spec/fixtures/v2/valid_10182_extra.json

```diff
@@ -19,7 +19,7 @@
           "phoneNumber": "8001111",
           "phoneNumberExt": "2"
         },
-        "email": "123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789_123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890123456789_12345678901234567890@bobbytablesemail.com",
+        "email": "bob@bobbytablesemail.com",
```

Comment:
Since you made these changes to the extra json file, you'll almost certainly also need to regenerate the `expected_10182_extra.pdf` to get all the tests to pass.